ui/src/common/icons/index.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 Open Reaction Database Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import CheckListIcon from './CheckListIcon.svg?react';
+import ChevronDownIcon from './ChevronDownIcon.svg?react';
+import CopyIcon from './CopyIcon.svg?react';
+import DotsIcon from './DotsIcon.svg?react';
+import DownloadIcon from './DownloadIcon.svg?react';
+import HomeIcon from './HomeIcon.svg?react';
+
+export { CheckListIcon, ChevronDownIcon, CopyIcon, DotsIcon, DownloadIcon, HomeIcon };
```

Comment:
We can use `export { default as CheckListIcon } from './CheckListIcon.svg?react';`     
We will need to only update one line when we add icons